lib/bolt11-tags.js

```diff
@@ -0,0 +1,10 @@
+import { decode } from 'bolt11'
+
+export function isBolt11 (request) {
+  return request.startsWith('lnbc') || request.startsWith('lntb') || request.startsWith('lntbs') || request.startsWith('lnbcrt')
+}
+
+export function bolt11Tags (bolt11) {
+  if (!isBolt11(bolt11)) throw new Error('not a bolt11 invoice')
+  return decode(bolt11).tagsObject
+}
```

Comment:
i've renamed `@/lib/bolt11` to `@/lib/bolt11-tags`

Reply:
Why not put everything bolt11 related into lib/bolt11.js? Same for bolt12 stuff?

Reply:
because ln-service uses 'fs' to load protobuf files  and it breaks webpack builds for the browser

Reply:
yes, it is possible, but last time dynamic imports caused a lot of issue in production.
Are they supposed to work reliably after your patch @huumn ?

Reply:
I don't have the full context but ...

I don't recall them causing issues in prod, but we should only use dynamic imports for bundle size reasons, or if the file we're including is determined dynamically. Using it so we can put code in `lib` that doesn't need to be there, seems silly to me.

If something is server specific, I recommend putting it in `api` and possibly creating an `api/lib` for future such code too.